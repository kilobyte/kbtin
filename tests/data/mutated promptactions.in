#pro abc #show first
#pro zxy {#show last;#end}
#loop 1,1000 {#pro b$0 #rem mid}
#pro b500 #unpro b*
#run a //selfpipe
b500
abc
zxy
