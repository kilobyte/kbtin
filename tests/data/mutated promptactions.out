b500
first
abc
last

#SESSION 'a' DIED.
